Typos in users.md, Meeting, Credits, Timeline

TheWarior73 opened the pull request:
ajout de certaines notes de traductions me semblant necessaires.  j'ai review le fichier, il se peut que des typo soient encore la.

TheWarior73 committed:
typo in users.md + review

ajout de certaines notes de traductions me semblant necessaires. 
j'ai review le fichier, il se peut que des typo soient encore la.

TheWarior73 committed:
Update README.md

typos

TheWarior73 committed:
Update README.md

TheWarior73 committed:
Update 2023-06-08-meeting.md

Fix des typo plus avancée dans le fichier

TheWarior73 committed:
oops 

🤡

TheWarior73 committed:
2023-06-08-meeting.md finit

j'ai finit la traduction du fichier, merci de le review, de virer les incohérences et de vérifier le français (j'ecrit vraiment mal désolé)

TheWarior73 commented:
j'ai normalement finit `2023-06-08-meeting.md` il faut vérifier derière moi, il y a 100% de chance que j'ai foutu une connerie dedans et je me sais incapable de la trouver.

TheWarior73 committed:
Ajout credits.md

Ajout des crédits, j'ai hésité a ajouter la team de traduction. A voir

TheWarior73 committed:
Create timeline.md

création de timeline.md

fantomitechno commented:
Promis, cette fois, demain je suis de retour, review et merge si c'est good

(J'ai regardé un peu les commits, ça fait toujours autant de mal de voir que j'écris si mal 😭 et je vous remercie de corriger ma connerie)

fantomitechno commented on lang/fr-FR/docs/users.md:
```diff
@@ -174,12 +176,12 @@ Le fait qu'aucun mod n'ait été infecté est un coup de chance.
 
 ### Comment CurseForge n'ont pas pu détecter cela ?
-Le code que l'Étape 0 exécute n'est pas nécessairement suspicieux pour un système de vérification automatisé, et pourrait très bien être quelque chose qu'un autre mod utilise. En faite, les premiers outils pour détecter l'Étape 0 renvoyaient un nombre signifiant de faux positifs sur des mods populaires tel que Quark.
+Le code de l'Étape 0 exécute n'est pas nécessairement suspicieux pour un système de vérification automatisé, et pourrait très bien être quelque chose qu'un autre mod utilise (une dépendance). En faite, les premiers outils pour détecter l'Étape 0 renvoyaient un nombre signifiant de faux positifs sur des mods populaires tel que Quark.
```
`Le code de l'Étape 0 exécute` ne veut rien dire, l'ancienne version faisait plus sens

fantomitechno commented on lang/fr-FR/docs/users.md:
```diff
@@ -1,14 +1,14 @@
 cela en suivant les instructions ci-dessous.
 
-Des informations additionnelles sur quoi faire si vous êtes ou non infecté seront données après les étapes permettant un diagnostic.
+(*translation note :* Des informations additionnelles sur quoi faire si vous êtes ou non infecté seront données après les étapes permettant un diagnostic.)
```
Ce n'est pas une note de traduction c'est (était peut-être, je n'ai pas suivi les update de l'anglais) sur le document anglais

TheWarior73 replied:
Je ne l'avait pas sur le document anglais mais c'est vrais que ça peut rester utile.

TheWarior73 commented:
j'ai corrigé ton écriture mais ça veut pas dire que ce que j'ai écrit est juste 😭

TheWarior73 committed:
Update users.md

fix des typo post review

TheWarior73 commented:
j'ai fix les elements indiqués dans la review. S'il y a quoi que ce soit d'autre...

fantomitechno commented:
Je regarde plus attentivement dès que je rentre 👍

mais de ce que j'ai vu (soit tout sauf le meeting.md) ça me semble good

TheWarior73 commented:
le meeting est la partie qui me fait le plus peur par rapport a ce que j'ai marqué, vas y avoir du travail a faire

TheWarior73 committed:
Merge branch 'timeline-and-other-stuff' into patch-2

fantomitechno commented:
@TheWarior73 tu as maintenant accès au repo

TheWarior73 committed:
Create tech.md

Création de tech.md
Traduit jusqu'a la ligne 232 (confondue au fichier français et anglais)

TheWarior73 commented:
Merci ! Je travaille sur `tech.md,` j'ai fait environ les 2/3 du fichier. (il faut que je relise le tout après coup)
je reviendrait sur `meeting` et `timeline` après pour safe check ce que j'ai écrit. Si jamais vous souhaitez jeter un coup d'oeil j'ai une branche WIP pour `tech.md` sur mon repo

TheWarior73 committed:
Update tech.md

I finished tech.md it is now up to typo fix and content review after doing some funky stuff with pull requests !

TheWarior73 committed:
Update tech.md

TheWarior73 committed:
Merge pull request #1 from TheWarior73/tech.md-WIP

Tech.md wip

TheWarior73 commented:
j'ai finit tech.md je review les typo demain pour tout mes fichiers (ceux que j'ai traduit), pour le contenu je vous laisse voir étant donné que ma traduction peut être incorrecte ! 😭

TheWarior73 committed:
Links, Typo fixes + last translations

Links fixed in evrery doc here,
Typos are still here but a litle bit less
I also added a disclaimer towards my translation of technical vocabulary in many files. This is probably the last patch for this pull request.

TheWarior73 committed:
Merge branch 'fantomitechno:main' into patch-2
